scripts/sns/install/upgrade_test_canister.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# For detailed explanation please see the GitHub Repo: https://github.com/dfinity/sns-testing/tree/main
+
+set -euo pipefail
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
```

Comment:
```suggestion
SCRIPT_DIR=$(dirname -- "${BASH_SOURCE[0]}")
cd  SCRIPT_DIR
```
this would be more readable.